File: laminatoria-client/src/app/classes/order.ts
import { Contacts } from './contacts'
import { OrderItem } from './orderItem'

export class Order {
	id: number
	contacts: Contacts
	status: number
	address: string
	comments: string
	date: Date
	orderItems: OrderItem[]
	delivery: string
	summary: number

	constructor(
		id: number,
		contacts: Contacts,
		status: number,
		address: string,
		comments: string,
		date: Date,
		orderItems: OrderItem[],
		delivery: string,
		summary: number
	) {
		this.id = id
		this.contacts = contacts
		this.status = status
		this.address = address
		this.comments = comments
		this.date = date
		this.orderItems = orderItems
		this.delivery = delivery
		this.summary = summary
	}
}

File: Laminatoria/Models/Order.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Laminatoria.Models
{
    public class Order
    {
        public int Id { get; set; }
        public Contact? Contacts { get; set; }
        public string Address { get; set; }
        public string Comments { get; set; }
        public DateTime Date { get; set; }
        public string Delivery { get; set; }
        [Column(TypeName = "decimal(9,2")]
        public decimal Summary { get; set; }
        public List<OrderItem> OrderItems { get; set; } = new List<OrderItem>();
    }
}
